README.md

```diff
@@ -69,60 +71,98 @@ This section explains how to install the latest stable release of
 
-- Using the [Anaconda Distribution](http://continuum.io/downloads).
-- Through [MacPorts](http://www.macports.org/).
+**Windows**
 
-### On GNU/Linux
+Spyder is included in these other scientific Python distributions:
 
-- Through your distribution package manager (i.e. `apt-get`, `yum`,
-  etc).
-- Using the [Anaconda Distribution](http://continuum.io/downloads).
-- Installing from source (see below).
+* [WinPython](https://winpython.github.io/)
+* [Python(x,y)](http://python-xy.github.io)
 
-### Cross-platform way from source
+**macOS**
 
-You can also install Spyder with the `pip` package manager, which comes by
-default with most Python installations. For that you need to use the
-command:
+Spyder can be obtained through through
+[MacPorts](http://www.macports.org/).
 
-    pip install spyder
+**GNU/Linux**
 
-To upgrade Spyder to its latest version, if it was installed before, you need
-to run
+* You can often get Spyder through your distribution's package
+manager (i.e. `apt-get`, `yum`, etc), or install from source
+(see below).
 
-    pip install --upgrade spyder
+### The Expert/Cross-platform Way: Installing from source with pip
 
-For more details on supported platforms, please refer to our
-[installation instructions](http://pythonhosted.org/spyder/installation.html).
+**Warning:** While this installation method is a viable option for
+experienced users, installing Spyder (and other SciPy stack packages)
+with `pip` can encounter a number of tricky issues. While you are welcome
+to try this on your own if you're an expert, we are unable to help if you
+do run into problems, except to recommend using Anaconda instead.
+
+You can also install Spyder with the `pip` package manager, which comes by
+default with most Python installations. To do so, you need to use the command:
+```
+    pip install spyder
+```
+To upgrade Spyder to its latest version, if it was installed before, you need
+to run:
+```
+    pip install --upgrade spyder
+```
 
 **Important note**: This does not install the graphical Python libraries (i.e.
 PyQt5 or PyQt4) that Spyder depends on. Those have to be installed separately
 after installing Python.
 
 
-## Running from source
+For more details on supported platforms, please refer to our
+[installation instructions](http://pythonhosted.org/spyder/installation.html).
 
-The fastest way to run Spyder is to get the source code using git, install
-PyQt5 or PyQt4, and run these commands:
 
-1. Install our *runtime dependencies* (see below).
-2. `cd /your/spyder/git-clone`
-3. `python bootstrap.py`
+## Running from source
 
+The fastest way to run Spyder is to run form source, hosted on the
+[Spyder github repo](https://github.com/spyder-ide/spyder).
 You may want to do this for fixing bugs in Spyder, adding new
 features, learning how Spyder works or just getting a taste of it.
+Make sure to copy the path listed under the "Clone or Download" button there,
+or just use https://github.com/spyder-ide/spyder.git .
```

Comment:
Let's remove the last two lines here to simplify the instructions below.

Reply:
Already done, @jnsebgosselin thought the same

Reply:
Agreed, removed completely.